pom.xml

```diff
@@ -242,6 +242,7 @@
                     <verbose>true</verbose>
                     <targetClasses>
                         <param>edu.ucsb.cs156.*</param>
+                        <!-- <param>edu.ucsb.cs156.organic.controllers.UsersController</param> -->
```

Comment:
This is commented out code.  Please see: 
https://ucsb-cs156.github.io/topics/code_reviews/commented_out_code.html

Reply:
Thank you! I have removed the code.